pkg/api/v1beta3/types.go

```diff
@@ -595,48 +593,48 @@ type PodStatusResult struct {
 // to hosts.
 type Pod struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"metadata specific to the Pod"`
```

Comment:
Actually, all of the metadata fields provide identical information. The intention is that it is standardized across all kinds of objects.

```
description:"standard object metadata; see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#metadata"
```